cluster/gce/windows/k8s-node-setup.psm1

```diff
@@ -1143,9 +1143,16 @@ function Start-WorkerServices {
   $kubelet_args_str = ${kube_env}['KUBELET_ARGS']
   $kubelet_args = $kubelet_args_str.Split(" ")
   Log-Output "kubelet_args from metadata: ${kubelet_args}"
+
+  # Windows nodes are renamed to a name <= 15 characters to join a AD domain.
```

Comment:
"are renamed" is vague. Can you be a little more specific in this comment: who/what does the renaming, and when, and under what conditions? Does the rename happen to the hostname at the Windows level, or to the instance name in GCE, or both?